app/services/ApprovalService.scala

```diff
@@ -66,4 +68,22 @@ class ApprovalService @Inject() (repository: ApprovalRepository) {
       case Right(success) => Right(Json.toJson(success).as[JsArray])
     }
   }
+
+  def approval2iReviewInfo(id: String): Future[RequestOutcome[JsObject]] = {
+    def mockData(): ApprovalProcessReview = {
+      ApprovalProcessReview(
+        "oct9005",
+        "Telling HMRC about extra income",
+        LocalDate.of(2020, 5, 10),
+        List(PageReview("id1", "how-did-you-earn-extra-income", "NotStarted"),
+          PageReview("id2", "sold-goods-or-services/did-you-only-sell-personal-possessions", "NotStarted"),
+          PageReview("id3", "sold-goods-or-services/have-you-made-a-profit-of-6000-or-more", "NotStarted"),
+          PageReview("id4", "sold-goods-or-services/have-you-made-1000-or-more", "NotStarted"),
+          PageReview("id5", "sold-goods-or-services/you-do-not-need-to-tell-hmrc", "NotStarted"),
+          PageReview("id6", "rent-a-property/do-you-receive-any-income", "NotStarted"),
+          PageReview("id7", "rent-a-property/have-you-rented-out-a-room", "NotStarted")))
+    }
+    Future(Right(Json.toJson(mockData()).as[JsObject]))
+  }
```

Comment:
Can this new function be moved into a `ReviewService`?

Reply:
Done - will be in next push